cmd/fleetctl/convert_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+func TestConvertFileOutput(t *testing.T) {
```

Comment:
Please default to using `t.Parallel()` except in tests that modify the environment variables or perform other side effects that prevent them from running in parallel.
```suggestion
func TestConvertFileOutput(t *testing.T) {
t.Parallel()
```